draft-ietf-ohai-svcb-config.md

```diff
@@ -131,9 +131,11 @@ The "ohttp" parameter can be included in the mandatory parameter
 list to ensure that clients that do not support access via Oblivious HTTP
 do not try to use the service. Services that mark the "ohttp"
 parameter as mandatory can, therefore, indicate that the service might
-not be accessible except through Oblivious HTTP. Services that optionally
-support access using Oblivious HTTP SHOULD NOT mark the "ohttp" parameter
-as mandatory. Note that since multiple SVCB responses can be provided for
+not be accessible except through Oblivious HTTP. Services that
+support access both with and without Oblivious HTTP SHOULD NOT mark 
```

Comment:
Any specific reason not to use "MUST NOT" ?
I can't think of a valid exception why "ohttp" parameter would be set to mandatory when it the service is accessible with and without Oblivious HTTP.